0015-automatic-cluster-upgrades/0015-automatic-cluster-upgrades.md

```diff
@@ -0,0 +1,139 @@
+# RFC 0015 - Automatic Cluster Upgrades
+
+## User stories
+
+- As a user, and also as KaaS team provider, I would like to have the clusters running in the latest version automatically, and not worry about CVEs or EOL of the cluster components.
```

Comment:
I'm not sure what `KaaS team provider` means. If it means a Giant Swarm KaaS provider team, I'm not sure it is necessary to encode that in an RFC as a stakeholder.

Also I'd try to avoid abbreviations in favour of simplicity and accessibility of the content.

```suggestion
- As a Giant Swarm customer, and also as Giant Swarm, we would like to have workload clusters running in the latest version automatically, and not worry about patching security vulnerabilities or cluster components running out of updates (e. g. Kubernetes release end of life).
```